app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt

```diff
@@ -111,6 +105,8 @@ class SignUpFragment : Fragment() {
 
     private fun redirectToMain() {
         val intent = Intent(activity, MainActivity::class.java)
+        intent.putExtra("isSnackbar", true)
```

Comment:
Unneeded

Reply:
I put this line to check whether the mainActivity is called from signupfragment and only in that  case show snackbar in mainActivity but if I remove this and also the check present in mainActivity  snackbar will be displayed every time we visit mainActivity. Pls correct

Reply:
Why though? It will only be displayed if the string to be displayed is passed in